micro_dl/inference/image_inference.py

```diff
@@ -784,7 +849,7 @@ def predict_2d(self, chan_slice_meta):
         target_stack = np.concatenate(target_stack, axis=0)
         # Stack images and transpose (metrics assumes cyxz format)
         if self.image_format == 'zyx':
-            if self.input_depth > 1:
+            if self.input_depth > 0:
```

Comment:
It looks like the default input_depth is 1. Is it ever <1? If not, do you even need the if statement?